lib/helpers/is-interactive-element.js

```diff
@@ -11,7 +11,9 @@ const INTERACTIVE_TAG_NAMES = [
   'keygen',
   'label',
   'select',
-  'textarea'
+  'textarea',
+  'audio',
```

Comment:
Please remove from `INTERACTIVE_TAG_NAMES` and instead create a helper method like:

```js
function isAudioVideoWithControls(node) {
  return (node.tag === 'audio' || node.tag === 'video') && ast.hasAttribute(node, 'controls');
}
```

Then add a snippet to `reasons` function (just below) like the following:

```js
if (isAudioVideoWithControls(node)) {
  return `an <${node.tag}> element with the \`controls\` attribute`;
}
```